packages/net-stubbing/lib/external-types.ts

```diff
@@ -176,8 +244,8 @@ export interface RouteMatcherOptionsGeneric<S> extends RouteMatcherCompatOptions
    */
   https?: boolean
   /**
-   * Match against the request's HTTP method.
-   * @default 'GET'
+   * Match against the request's HTTP or WebDAV method.
```

Comment:
```suggestion
   * Match against the request's HTTP method.
```

Let's keep it simple, 99% of users won't be using WebDAV. Plus, I kinda feel like if it's in the HTTP header's method field, it's an HTTP method, so :shrug: